stories/imdb/stories/greyVest.js

```diff
@@ -339,7 +340,10 @@ export default () => (
                 <div className="gv-box">
                   <ResultTable
                     path={['root', 'results']}
-                    fields={schemas[tree.tree.schema].fields}
+                    fields={_.omit(
+                      ['imdbId'],
```

Comment:
we should validate this further with a test case that has at least two fields omitted to prove it works

Reply:
Fixed